src/runtime.cpp

```diff
@@ -448,3 +448,19 @@ extern "C" int dlltmatsolve(void** solverPtr,
                       Bn, Bm, Browptr, Bcolidx, Bnn, Bmm, Bvals,
                       Xn, Xm, Xrowptr, Xcolidx, Xnn, Xmm, Xvals);
 }
+
+/// cross product between 2 vectors3D
+template <typename Float>
+int cross(int an, Float* a, int bn, Float* b, int cn, Float* c){
```

Comment:
I get a warning here since the `int cross` function does not return anything. Could you make the `cross` functions void? (They are not going to fail.)